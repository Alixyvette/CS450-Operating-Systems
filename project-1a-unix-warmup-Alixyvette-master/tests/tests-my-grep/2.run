./my-grep this INPUT_DIR/2.in
