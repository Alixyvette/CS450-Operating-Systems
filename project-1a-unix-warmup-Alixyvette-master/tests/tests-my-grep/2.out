my-grep: cannot open file
